app/CocoaPods/Base.lproj/MainMenu.xib

```diff
@@ -86,6 +86,11 @@
                     <modifierMask key="keyEquivalentModifierMask"/>
                     <menu key="submenu" title="File" id="bib-Uj-vzu">
                         <items>
+                            <menuItem title="New Podfile" keyEquivalent="n" id="Q1o-LS-sQI">
```

Comment:
OK, can you rename this to "New Podfile from Xcodeproj" ? - then this is good to merge :+1: